app.js

```diff
@@ -16,13 +21,10 @@ db.once('open', function() {
   });
 });
 
+const xxx = db.collection('users')
 
-const express = require('express')
-const app = express()
-const port = process.env.PORT || 8080;
-app.get('/', function (req, res) {
-  res.send('Hello World')
+app.get('/', async function (req, res) {
```

Comment:
Lovely async await so that we can give the database time to retrieve what we asked for and show it on our route